maxim/models/dataset.py

```
@@ -27,8 +27,16 @@ class Variable:
     type: Literal["text", "json", "file"]
     payload: Union[str, dict[str, Any], list[Attachment]]
 
+    def __init__(self, type: Literal["text", "json", "file"], payload: Union[str, dict[str, Any], list[Attachment]]):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)